dask/blockwise.py

```diff
@@ -17,6 +22,20 @@
 )
 
 
+def blockwise_generate_chunk(idx, chunks):
+    """Generate a local dask.array chunk"""
```

Comment:
not a fan of the name - this is a generic module, not specific to dask.array, but there's nothing in the name of this function that says this is a dask.array-specific function. Also, the function name is mismatched with the string label "generate_array_chunk"